components/section/Footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PageLink from '../ui/Link'
+import Svg from '../ui/Svg'
+import Wrapper from '../wrappers/Wrapper'
+
+export default function Footer() {
+    return (
+       <Wrapper>
+         <div className="pb-6.1563 md:pb-14.8125 lg:pb-4.8356 xl:pb-4.4306 2xl:pb-3.0556 md:flex space-y-14 md:items-baseline md:justify-between">
+            <div className="w-11.625 md:mt-auto md:w-13.875 font-Neufile font-normal text-xl md:text-2xl text-black tracking-0.0125">
```

Comment:
you can try to use like a span inside to display the text contents or use a title instead of a div